### DISCUSSION TITLE
How to add "notification"

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#2598] [Q&A] [answered]
Hi,
Can someone tell me what is the best way to add a notification feature in my pocketbase.

I have the collections "posts" that has a relation to the "users" collection.

Other users can upvote,downvote and comment the post of others. I want to add now a notification that saves milestones of the post or that a user commented on their post. Should the notification be an own collection or is there a simple and efficient way to do this.

![Screenshot from 2023-05-30 12-23-50](https://github.com/pocketbase/pocketbase/assets/22890183/78e1b600-dfcd-4493-b0d3-fc734edc0275)

Any help is much much appreciated!

**Comment by Lex-2008** [comment, marked as answer]
this looks very much like https://github.com/pocketbase/pocketbase/discussions/2235. I believe you could extend the `alreadyVoted` collection to add a "type" of "comment" and with a text field of "comment text". Then, PocketBase automatically adds "created" field to each record. This way, by making one request like this:
```
await locals.pb.collection("alreadyVoted").getFullList( {
    filter: `post="${postId}"`, sort: 'created',
})
```
you will get list of all comments and votes, in the correct order!

**Reply by MuslemRahimi** [reply]
thank you for the answer :+1:
